### ISSUE TITLE
GetPackageOriginalFile sudo /etc/sudoers): No such file or directory

### ISSUE THREAD

**Issue by jfchevrette** [CyberShadow/aconfmgr#69]
This is on a freshly installed system.

```
::: Sourcing /home/jfchevrette/.config/aconfmgr/30-base.sh...
:::: Downloading package sudo (sudo-1.8.30-1-x86_64.pkg.tar.xz) to pacman's cache
::::: Proceed? [Y/n]


Package (1)  Old Version  New Version  Net Change

core/sudo    1.8.30-1     1.8.30-1       0.00 MiB

Total Download Size:  0.00 MiB

:: Proceed with download? [Y/n]
(1/1) checking keys in keyring                                                                                                 [-----------------------------------------------------------------------------] 100%
(1/1) checking package integrity                                                                                               [-----------------------------------------------------------------------------] 100%
::::: Done.
:::: Unable to find package file for package sudo!
:::: Fatal error! Stack trace:
::::: /usr/lib/aconfmgr/common.bash:1379 [AconfGetPackageOriginalFile]
::::: /usr/lib/aconfmgr/helpers.bash:185 [GetPackageOriginalFile]
::::: /home/jfchevrette/.config/aconfmgr/30-base.sh:37 [source]
::::: /usr/lib/aconfmgr/common.bash:112 [AconfCompileOutput]
::::: /usr/lib/aconfmgr/common.bash:779 [AconfCompile]
::::: /usr/lib/aconfmgr/apply.bash:168 [AconfApply]
::::: /usr/lib/aconfmgr/main.bash:163 [Main]
::::: /usr/lib/aconfmgr/main.bash:177 [source]
::::: /usr/bin/aconfmgr:26 [main]
/home/jfchevrette/.config/aconfmgr/30-base.sh: line 37: $(GetPackageOriginalFile sudo /etc/sudoers): No such file or directory
:::: Fatal error! Stack trace:
::::: /home/jfchevrette/.config/aconfmgr/30-base.sh:37 [source]
::::: /usr/lib/aconfmgr/common.bash:112 [AconfCompileOutput]
::::: /usr/lib/aconfmgr/common.bash:779 [AconfCompile]
::::: /usr/lib/aconfmgr/apply.bash:168 [AconfApply]
::::: /usr/lib/aconfmgr/main.bash:163 [Main]
::::: /usr/lib/aconfmgr/main.bash:177 [source]
::::: /usr/bin/aconfmgr:26 [main]
```

When I look in my local pacman cache, I see a file with a different extension for the sudo package: `/var/cache/pacman/pkg/sudo-1.8.30-1-x86_64.pkg.tar.zst`. Could this be why I'm having this error?

**Comment by jfchevrette** [comment]
Fixed by 1176b833bd8dcbdbc4e7b7a5022c697f8fd0e3ca

**Comment by CyberShadow** [comment]
Thanks for the report and for testing the fix :) I got a bit sidetracked with fixing follow-up CI issues.
